config/routes.rb

```diff
@@ -1,2 +1,3 @@
 ShiftCommerce::Engine.routes.draw do
+  get "static_pages/:id", to: "static_pages#show", as: :pages
```

Comment:
Moved from F/E without changes – does not currently respect RackSlug, as it has not yet been extracted.